index copy.html

```diff
@@ -0,0 +1,385 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>The Portfolio of Massaillon Home page</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/8ea4cd9e84.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header id="header-container">
+    <div class="logo-container">
+        <a href="#">Welcome</a>
+    </div>
+
+    <nav class="navbar">
+        <div class="underline-bar-navmenu hide"></div>
+        <ul class="nav-menu hide">
+            <li class="nav-item">
+                <a  href="#main-home" class="nav-link">Portfolio</a>
+            </li>
+            <li class="nav-item">
+                <a href="#About-Me" class="nav-link">About</a>
+            </li>
+            <li class="nav-item">
+                <a href="#form" class="nav-link" >Contact</a>
+            </li>
+        </ul>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+            
+        </div>
```

Comment:
- [ ] If you see the Figma design then the `X` buttons are not similar to the design kindly make the design similar with the Figma 
### your design
![his](https://user-images.githubusercontent.com/81727988/168335589-330c3147-a0f7-46e7-ae58-119a9a6bea5c.png)
### Figma design

![figma](https://user-images.githubusercontent.com/81727988/168335756-63144970-d7a4-443d-8c54-ce130309bbfe.png)